source/gent/tier1_hortense.rst

```diff
@@ -216,28 +221,100 @@ This storage space can be addressed with the environment variable $TMPDIR
   cd $TMPDIR
 
 
-Project storage
-***************
+.. _hortense_home_on_scratch:
+
+Home-on-scratch setup
+*********************
+
+On Tier-1 Hortense, the home directory (``$HOME``) corresponds to your personal scratch directory (``$VSC_SCRATCH``),
+rather than your usual VSC home directory (``$VSC_HOME``).
+
+This is done to ensure that Tier-1 Hortense can remain operational, even if there is maintenance being
+performed on the Tier-2 shared storage filesystem of a VSC site (UGent, KUL, VUB, UAntwerpen),
+or in case of problems with the network connection to the other VSC sites.
 
-* Please be aware that storage space on $VSC_SCRATCH is limited per user to 3 GB.
-* Instead, it is better to use the dedicated scratch storage space which is reserved for every Tier1 project.
-* The environment variable $VSC_SCRATCH_PROJECTS_BASE points to the basefolder containing all project spaces.
-* Project space is given the same name as your project.
-* You can e.g. use this command to directly switch to your dedicated project space:
+Although your VSC home directory is usually accessible via ``$VSC_HOME``,
+we strongly recommend to *not* simply create symbolic links to files like your ``.bashrc`` startup script,
+since that would defeat the purpose of this "home-on-scratch" setup.
+
+This recommendation also applies to ``$VSC_DATA``: you should avoid using it in your job scripts as much as
+possible, and ensure that your workflow only relies on the Hortense scratch filesystem. If you require any
+data as input for your jobs, it should be copied to the Hortense scratch filesystem first.
+
+.. _hortense_project_scratch_dirs:
+
+Project scratch directories
+***************************
+
+* Please be aware that storage space on ``$VSC_SCRATCH`` (personal scratch directory) is limited per user to 3 GB.
+* Instead, it is better to use the dedicated scratch storage space which is reserved for your Tier-1 project.
+* The environment variable ``$VSC_SCRATCH_PROJECTS_BASE`` points to the base folder containing all project directories.
+* Project directories are given the same name as your Tier-1 project (so *without* a prefix like ``gpr_compute_``).
```

Comment:
@epauwels what other prefixes do we have : startin and explor?

Reply:
I'm happy to fix the docs to mention all possible prefixes, but not in this PR please :)